dev_menu.py

```diff
@@ -132,8 +132,24 @@ def provision_virtualenv(venv_path=DEFAULT_PYENV):
     ('[Local] Python Unit tests',
         "./py3_venv/bin/nosetests -v tests/python/unittest/"
     ),
-    ('[Docker] Website and docs build outputs to "docs/_build/html/"',
-        "ci/build.py --platform ubuntu_cpu /work/runtime_functions.sh deploy_docs"),
+    ('[Docker] Build the MXNet binary - outputs to "lib/"',
```

Comment:
mxnet binary? not clear what this does.

Reply:
It builds the .so and .a file that the docs need for API docs generation.

Reply:
only the .so file is needed AFAIK